gandalf/src/main/java/io/github/btkelly/gandalf/Callback.java

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright 2016 Bryan Kelly
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.github.btkelly.gandalf;
+
+import io.github.btkelly.gandalf.models.Alert;
+import io.github.btkelly.gandalf.models.OptionalUpdate;
+import io.github.btkelly.gandalf.models.RequiredUpdate;
+
+public interface Callback {
```

Comment:
I'm open to anything as far as a name, I think I just wasn't feeling creative when I made the class. And I don't have any ideas at the very moment. I suppose I'll sleep on it and see if I dream up a better name.